platform-sdk/consensus-otter-tests/src/testFixtures/java/org/hiero/otter/fixtures/logging/internal/InMemoryAppender.java

```diff
@@ -95,6 +95,18 @@ public static List<StructuredLog> getLogs() {
         return Collections.unmodifiableList(logs);
     }
 
+    /**
+     * Returns an unmodifiable list of all captured log statements for a specific node
+     *
+     * @param nodeId nodeId of the node
+     *
+     * @return an unmodifiable list of all captured log statements for {@code nodeId}
+     */
+    @NonNull
+    public static List<StructuredLog> getLogs(final long nodeId) {
+        return logs.stream().filter(l -> l.nodeId() == nodeId).toList();
```

Comment:
Please never use `l` as a variable name. I remember spending a whole afternoon debugging some code with a colleague just to find out that a single `1` was used in one place. We did not see it even after staring at the screen for hours... 😂

Reply:
changed all `l` :D